### DISCUSSION TITLE
Configuring WireGuard Easy to Use LAN IP Addresses

### DISCUSSION THREAD

**Discussion by IoSonoPiero** [wg-easy/wg-easy#1401] [Q&A] [answered]
I'm currently using WireGuard Easy with Docker.
The setup works well, and it correctly generates client data with addresses in the 10.8.0.x range sequentially (10.8.0.1, 10.8.0.2, etc.).

My home network setup is as follows:
- Router assigns addresses in the 192.168.1.x range to local devices
- The server running Docker has the IP address 192.168.1.100

## The Challenge

I want to configure WireGuard Easy to generate addresses in the 192.168.1.x range, specifically starting from 192.168.1.200 and incrementing (192.168.1.201, 192.168.1.202, etc.). This would allow me to connect to my LAN and access my local devices remotely using WireGuard.

## Questions

1. Is it possible to configure WireGuard Easy to use this IP range?
2. If so, what changes do I need to make to the Docker configuration or WireGuard Easy settings?
3. Are there any potential conflicts or issues I should know when using LAN IP addresses for WireGuard clients?

Any guidance or solutions would be greatly appreciated.
Thank you!

**Comment by kaaax0815** [comment]
You should be able to access Local Services even if you have a different IP. As the Local IP of the Server would be used to make a connection to the local services. 

You can change `WG_DEFAULT_ADDRESS` to `192.168.1.x` which would generate Clients in the correct subnet. But your router probably lives on `192.168.1.1` where wg-easy also wants to live on

**Reply by IoSonoPiero** [reply]
Hello, and thank you for the answer.

I cannot access Local Services when connected to Wireguard when I'm not at home. And this is crazy.

Changing `WG_DEFAULT_ADDRESS` to `192.168.1.x` creates clients starting from `192.168.1.1`, so it's unsuitable for me.
For this reason, I opened a ticket here.

**Reply by kaaax0815** [reply]
With #1347 you have a workaround by using a non standard cidr

**Reply by latimeria68** [reply]
> Hello, and thank you for the answer.
> 
> I cannot access Local Services when connected to Wireguard when I'm not at home. And this is crazy.
> 
> Changing `WG_DEFAULT_ADDRESS` to `192.168.1.x` creates clients starting from `192.168.1.1`, so it's unsuitable for me. For this reason, I opened a ticket here.

Did you try to put 192.168.1.0/24 in the allowed ips? This way you should be able to access machines on your Lan, no need to play with WG default address env

**Reply by IoSonoPiero** [reply]
Currently, I have this:
`WG_ALLOWED_IPS=192.168.1.0/24, 0.0.0.0/0, ::/0`

**Reply by IoSonoPiero** [reply, marked as answer]
IT'S WORKING!!!
Guys, really.
I was not able to connect to remote devices.
Now I can. I've not changed a line.

I don't know what the issue was or what it was.

Currently, I'm connected to a router with a SIM, and I'm not home.
I can access all my devices.

Sorry for my post.

**Reply by clowgg** [reply]
@IoSonoPiero Did you figure out how to limit the IP addresses that W/G uses when it configures each remote client config file/QR Code?

**Reply by IoSonoPiero** [reply]
Hi @clowgg,
Actually, I didn’t configure a specific IP range for remote clients—it just started working for me without any changes. So, unfortunately, I don’t have a solution for limiting the IP addresses that WireGuard assigns when creating each client config/QR code.
If you find a way, please let me know too! Thanks.

**Reply by kaaax0815** [reply]
@clowgg Do you mean `WG_DEFAULT_ADDRESS` in v14, the cidr in v15?

**Reply by clowgg** [reply]
I'm sorry, I don't know what the item is @kaaax0815. 
I just know I need to avoid W/G choosing to configure remote clients with IP addresses that are already in use on the LAN.

**Reply by kaaax0815** [reply]
you can do that via the change cidr option in the admin panel in v15
